src/sentry/models/groupforecast.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    ArrayField,
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+)
+
+
+@region_silo_only_model
+class GroupForecast(Model):
+    """
+    Stores the forecast of expected counts of events for a Group
+
+    ``forecast`` will hold an array of integers with
+    a length of 8 (the difference between date_created and valid_until)
+
+    """
+
+    __include_in_export__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True, db_constraint=False)
+    project = FlexibleForeignKey("sentry.Project", null=True, db_constraint=False)
+    organization = FlexibleForeignKey("sentry.Organization", null=True, db_constraint=False)
```

Comment:
You don't have to set `db_constraint=False` here if you don't want to, it'll be fine to add constraints to these tables.